hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/AbstractAttributeEnricherTest.java

```diff
@@ -179,16 +179,13 @@ protected AttributeValue createAvroAttribute(String value) {
   }
 
   Event.Builder createOpenSourceSpan(String tenantId, String eventId, String jaegerService) {
-    Map<String, AttributeValue> rawAttributeMap = Map.of(
-        Constants.getRawSpanConstant(JaegerAttribute.JAEGER_ATTRIBUTE_SERVICE_NAME),
-        AttributeValue.newBuilder().setValue(jaegerService).build()
-    );
-
+    Map<String, AttributeValue> rawAttributeMap = Map.of();
```

Comment:
Can we get rid of this map, since it's empty now? No need to set empty attributes on the Event